### DISCUSSION TITLE
Consider label (scalar) and labels (string array) in info response

### DISCUSSION THREAD

**Discussion by jbfaden** [hapi-server/data-specification#144] [Ideas]
I see that you can either describe the vector "B-GSM" or you can describe each component ("BX-GSM", "BY-GSM" and "BZ-GSM") but not both.  Often times you want both, and we should consider having a separate "labels" parameter attribute which is the array.  This would also allow the label to have just one type (string) so that clients don't have to check the type and probably makes validation easier too.

**Comment by rweigel** [comment]
It seems like we already have this; [`label`](https://github.com/hapi-server/data-specification/blob/master/hapi-dev/HAPI-data-access-spec-dev.md#364-parameter-object) can be a string or an array of strings.

**Comment by jbfaden** [comment]
But I want both.  Another example might be label="instrument states" and labels=['inst1','inst2','inst3'].  Or label='complex' and labels=['real','imag'].  I'm sure we had a discussion about this, but I don't remember why having separate names for two different things is not allowed.  I'm bringing this up because I've been looking for an example of a vector with labelled components, and people aren't doing this.
